public/docs/_examples/animations/ts/app/hero-team-builder.component.ts

```diff
@@ -0,0 +1,104 @@
+import { Component } from '@angular/core';
+import { Hero, Heroes } from './hero.service';
+import { HeroListBasicComponent } from './hero-list-basic.component';
+import { HeroListInlineStylesComponent } from './hero-list-inline-styles.component';
+import { HeroListEnterLeaveComponent } from './hero-list-enter-leave.component';
+import { HeroListEnterLeaveStatesComponent } from './hero-list-enter-leave-states.component';
+import { HeroListCombinedTransitionsComponent } from './hero-list-combined-transitions.component';
+import { HeroListTwowayComponent } from './hero-list-twoway.component';
+import { HeroListAutoComponent } from './hero-list-auto.component';
+// Enable when CSS parser integration has been added
+// import {HeroListClassesComponent} from './hero-list-classes.component';
+import { HeroListGroupsComponent } from './hero-list-groups.component';
+// Enable when CSS parser integration has been added
+// import {HeroListKeyframesComponent} from './hero-list-keyframes.component';
+import { HeroListMultistepComponent } from './hero-list-multistep.component';
+import { HeroListTimingsComponent } from './hero-list-timings.component';
+
+@Component({
+  selector: 'hero-team-builder',
+  template: `
+    <button [disabled]="!heroes.canAdd()" (click)="heroes.add()">Add</button>
+    <button [disabled]="!heroes.canRemove()" (click)="heroes.remove()">Remove</button>
+    <div class="columns">
```

Comment:
See earlier comment about TOC to provide more information. Perhaps a column mouseOver to show details in the footer pane ?